napari/_qt/__init__.py

```diff
@@ -58,7 +58,7 @@
             deferred=True,
             version=QtCore.__version__,
         )
-    warn(message=warn_message)
```

Comment:
My understanding was that stacklevel does not work on import and ends up in the CPython import machinery.

I'm also unsure the error message would be useful as it would not point to an actual error the user is doing on the line of the import, but I can still put 2.